### PR TITLE
Support boot loop detection for configuration failures at boot time

### PR DESCRIPTION
**Pull request by ryanpetrello**
...DOWN.

### CODE REVIEW

File: doc/source/state_machine.dot [hunk 1]
```diff
@@ -32,8 +32,8 @@ digraph rug {
   CREATE_VM -> CALC_ACTION [ label = "vm:E" ];
 
   CHECK_BOOT -> CONFIG [ label = "vm>U" ];
-  CHECK_BOOT -> CALC_ACTION [ label = "vm:[DBCR]" ];
-  CHECK_BOOT -> STOP_VM [ label = "vm:G" ];
+  CHECK_BOOT -> CALC_ACTION [ label = "vm:[BCR]" ];
+  CHECK_BOOT -> STOP_VM [ label = "vm:[DG]" ];
```
Comment (dhellmann) [inline doc/source/state_machine.dot:36]: CheckBoot.transition() never returns StopVM().

File: akanda/rug/state.py [hunk 2]
```diff
@@ -234,7 +234,8 @@ def execute(self, action, worker_context):
         return action
 
     def transition(self, action, worker_context):
-        if self.vm.state == vm_manager.GONE:
+        if self.vm.state in (vm_manager.DOWN,
```
Comment (ryanpetrello) [inline akanda/rug/state.py:237]: Right here?

Comment (dhellmann) [inline akanda/rug/state.py:237]: Blah.

**Comment by ryanpetrello** [comment]
These changes will go in with Doug's work at https://github.com/dreamhost/akanda-rug/pull/162/commits

**Pull request closed without merging**